### ISSUE TITLE
Run on Azure windows VM remotely got error :  Message: An unknown error occurred in the remote end while processing the command.

### ISSUE THREAD

**Issue by dimag25** [microsoft/WinAppDriver#1799]
Hello, 
I've got the same issue that was previously posted : 
https://github.com/microsoft/WinAppDriver/issues/147

I'm trying to perform an install on .msi installation via the GUI remotely on azure windows 10 vm.
I've tried all the solutions from the post but I have the following error when I'm trying to perform clicks on the windows application.
when I'm running an RDP connection and then trying the script works.

the problem when I'm running in the CI server(Jenkins Linux server) I cannot perform the RDP connection and it not works.

tried the following solutions:
https://community.helpsystems.com/kb-nav/kb-article/?id=548d72cc-68c3-eb11-bacc-000d3a1fe4c0&redirect=false
 https://dev.to/beanworks/the-painful-parts-of-end-to-end-test-automation-for-your-windows-application-1dde

tried also:
[Psexec ](https://docs.microsoft.com/en-us/sysinternals/downloads/psexec)- to start the server on the remote machine console
[Autologon ](https://docs.microsoft.com/en-us/sysinternals/downloads/autologon)- to unlock the machine on startup

any suggestions and help?

**Comment by dimag25** [comment]
I mean the RDP workaround change didn't helped according to the following reg keys changes:
https://community.helpsystems.com/kb-nav/kb-article/?id=548d72cc-68c3-eb11-bacc-000d3a1fe4c0&redirect=false

the reg keys changed successfully but I got the same error, without opening the RDP connection by myself with full GUI.

is there any way to simulate opening RDP to win 10 from Jenkins's linux server?

**Comment by anunay1** [comment]
Can you setup teams call @anunay1 ? Not able to figure out what you are trying to achieve?

**Comment by dimag25** [comment]
@anunay1 , yes will try setup call, tomorrow morning around 8.00 UTC time is ok? 
 via Zoom is ok instead of teams?

**Comment by anunay1** [comment]
Zoom is fine but please schedule a call at 1PM UTC

**Comment by dimag25** [comment]
@anunay1  will you be available around 10-11, or after 15:00 (UTC time)?

**Comment by bwomsm1** [comment]
@dimag25 did you manage to solve the issue? 
Are you running against Appium server or `WinAppDriver` directly?

**Comment by dimag25** [comment]
hey @bwomsm1 , still not managed to solve the issue.
im running against  server with the following url :
f'http://{vm_address}:4723/wd/hub

I guess its appium server?

**Comment by bwomsm1** [comment]
Could be both I think ...

**Comment by dimag25** [comment]
ok.. so im uaing this server in the vm and starting the winappdriver with batch file LaunchWAD.cmd :

cmd start /K "C:/Program Files (x86)/Windows Application Driver/WinAppDriver.exe" 10.0.0.4 4723/wd/hub

**Comment by bwomsm1** [comment]
So it's the WinAppDriver directly

**Comment by dimag25** [comment]
ok, and changing it to use appium server will fix my issue?

**Comment by anunay1** [comment]
No it will not, appium also uses winappdriver.exe under the hood, what you are trying to do is to execute tests in a headless mode which is not possible for WAD as it requires an interactive session.

**Comment by bwomsm1** [comment]
@anunay1 please check this issue: https://github.com/appium/appium/issues/17572#issuecomment-1269834433
We have a major issue with `WinAppDriver.exe`, that we got to fix. 
I cannot downgrade my python libs, I must have methods which found only in the latest appium client libs. 

Can we estimate when this going to be handled? it's quite showstopper.

**Comment by anunay1** [comment]
@DHowett Can we expect any release?

**Comment by bwomsm1** [comment]
> I mostly encounter this issue when the PC is locked or else the buffer is full. There exists a bug with win app driver console at times the buffer is not cleared. As a work around i use to start winappdriver console before the every test and then kill it after the test.

That what I have started doing, I also starting it on every session

**Comment by bwomsm1** [comment]
> @DHowett Can we expect any release?

@DHowett @anunay1 can you share with me when that w3c will be supported?
